### PR TITLE
Feat/terra station

### PR DESCRIPTION
**Pull request by canhtrinh**
## Description
* merged latest from develop to previous work in this branch: https://github.com/axelarnetwork/axelar-web-app/tree/feat/add-terra-station-support
* adding warning for situation where network is not correct (i.e. Terra Station connected to testnet when it should be mainnet and vice versa)
* adding Terra Station autofill option for destination address
* updating Connect Wallet in header to modal options
* updating Terra Station wallet connect method to check for installation first. 

## Todos

- [ ] Unit tests
- [x] Manual tests
- [ ] Documentation
- [ ] Connect epics/issues
- [ ] Tag type of change

## Other Notes

### CODE REVIEW

File: src/pages/app/parts/swap-widget/TransactionStatusWindow/index.tsx [hunk 1]
```diff
@@ -157,17 +195,59 @@ const TransactionStatusWindow = ({
   const setShowLargeDisclaimer = useSetRecoilState(ShowLargeDisclaimer)
   const [userConfirmed, setUserconfirmed] = useState(false)
   const [walletToUse, setWalletToUse] = useState<WalletInterface | null>()
+  const terraWallet = useWallet()
+  const lcdClient = useLCDClient({
+    URL:
+      process.env.REACT_APP_STAGE === "mainnet"
+        ? "https://lcd.terra.dev"
+        : "https://bombay-lcd.terra.dev",
```
Comment (npty) [inline src/pages/app/parts/swap-widget/TransactionStatusWindow/index.tsx:203]: could replace the url string with `terraConfigMainnet` and `terraConfigTestnet` here

File: src/pages/app/parts/swap-widget/UserInputWindow/index.tsx [hunk 2]
```diff
@@ -387,6 +415,16 @@ const UserInputWindow = ({
...
+              {destChainSelection?.chainName?.toLowerCase() === "terra" && "OR"}
+              {destChainSelection?.chainName?.toLowerCase() === "terra" && (
+                <StyledSVGImage
+                  onClick={getDestinationAddressFromTerraWallet}
+                  height={`1.35em`}
+                  width={`1.35em`}
+                  margin={`0em 0.75em 0em 0.5em`}
+                  src={require(`assets/svg/terra-station.svg`).default}
+                />
+              )}
```
Comment (npty) [inline src/pages/app/parts/swap-widget/UserInputWindow/index.tsx:427]: can combine to something like below
```js
{destChainSelection?.chainName?.toLowerCase() === "terra" && (
  <>
      <span>"OR"</span>
      <StyledSVGImage
        onClick={getDestinationAddressFromTerraWallet}
        height={`1.35em`}
        width={`1.35em`}
        margin={`0em 0.75em 0em 0.5em`}
        src={require(`assets/svg/terra-station.svg`).default}
      />
  </>
  )}
  ```

File: src/pages/app/parts/swap-widget/TransactionStatusWindow/index.tsx (hunk 1 above)
Comment (canhtrinh) [reply src/pages/app/parts/swap-widget/TransactionStatusWindow/index.tsx:203]: done

File: src/pages/app/parts/swap-widget/UserInputWindow/index.tsx (hunk 2 above)
Comment (canhtrinh) [reply src/pages/app/parts/swap-widget/UserInputWindow/index.tsx]: also done

**Review by npty — APPROVED** [review]

**Pull request merged**